.circleci/config.yml

```diff
@@ -10,19 +10,19 @@ executors:
     working_directory: ~/repos/celo-monorepo/packages/celotool
     environment:
       GO_VERSION: "1.13.7"
-      CELO_MONOREPO_BRANCH_TO_TEST: master
+      CELO_MONOREPO_BRANCH_TO_TEST: kobigurk/fix_hash_to_group
```

Comment:
Can we be sure to fix up https://github.com/celo-org/celo-monorepo/pull/2977 before we merge this?

Reply:
Yeah if this CI passes, we'll merge the branch on the monorepo, point CI back to master and merge.